src/nyx/image_loader.cpp

```diff
@@ -41,7 +44,12 @@ bool ImageLoader::open(const std::string& int_fpath, const std::string& seg_fpat
 		{
 			if (Nyxus::check_tile_status(int_fpath))
 			{
-				intFL = new NyxusGrayscaleTiffTileLoader<uint32_t>(n_threads, int_fpath);
+				intFL = new NyxusGrayscaleTiffTileLoader<uint32_t> (

```

Comment:
Instead of using globals here, can we modify the open() to pass the `fimageOptions`?

Reply:
My motive to access the global settings as close to the point of use as possible, as just one control flow branch of many passes through the floating point pixels/voxels manipulation. Otherwise we would need to prepare those global settings to pass as parameters early amid the high-level code of phase 2 (trivials) and 3 (non-trivials) image processing logic where we are not concerned about byte layouts in pixels, and do it twice - in function processDataset() and processNontrivialRois(). Your opinion?